src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
```

`html` tag doesn't has a margin
Change it to `body` selector
```suggestion
body {
```